media/js/src/map.js

```diff
@@ -0,0 +1,33 @@
+requirejs(['./common'], function(common) {
+    const libs = ['jquery', 'bootstrap', 'Vue', 'vue2leaflet', 'utils'];
+    requirejs(libs,
+        function($, bootstrap, Vue, vue2leaflet, utils) {
+            new Vue({
+                el: '#content',
+                data: function() {
+                    return {
+                        zoom: 5,
+                        center: vue2leaflet.L.latLng(37.0902, -95.7129),
+                        url: 'http://{s}.tile.osm.org/{z}/{x}/{y}.png',
```

Comment:
Is it possible to make this link `https`?

Reply:
Yes, you’ll the update in a subsequent pr